doc/source/ray-core/runtime_env_auth.md

```diff
@@ -0,0 +1,120 @@
+(runtime-env-auth)=
+# Authenticating Remote URIs in your runtime_env
+
+This section helps you:
+
+* Avoid leaking remote URI credentials in your `runtime_env`
+* provide credentials safely in KubeRay
+* understand best practices when authenticating your remote URI
+
+## Authenticating Remote URIs
+
+You can add dependencies to your `runtime_env` with [remote URIs](remote-uris). This is straightfoward for files hosted publicly, since you can simply paste the public URI into your `runtime_env`:
```

Comment:
```suggestion
You can add dependencies to your `runtime_env` with [remote URIs](remote-uris). This is straightforward for files hosted publicly, because you simply paste the public URI into your `runtime_env`:
```